src/Sentry/Sentry.csproj

```diff
@@ -1,38 +1,41 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>net5.0;netcoreapp3.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
```

Comment:
We need to make sure this TFM is tested now.

i.e:

```
  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
    <ProjectReference Include="../../src/Sentry/Sentry.csproj" Properties="TargetFramework=netcoreapp3.0" />

and
  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
    <ProjectReference Include="../../src/Sentry/Sentry.csproj" Properties="TargetFramework=netstandard2.1" />

```

If there's a way to get netcoreapp3.1 to run twice, once against 1 target and once against the other, we're good.